lib/noted/notes.ex

```diff
@@ -290,12 +304,12 @@ defmodule Noted.Notes do
       {:error, %Ecto.Changeset{}}
 
   """
-  def create_tag!(attrs \\ %{}) do
+  def create_tag(attrs \\ %{}) do
```

Comment:
Has `create_tag!` been replaced everywhere that it is used by `create_tag`? If so, that's fine. Just don't know :)

Reply:
it has been replaced everywhere.